lib/output.rb

```diff
@@ -0,0 +1,58 @@
+class Output
+
+    attr_reader :draw_text, :player1_win, :player2_win
```

Comment:
Great job building this output class.  As you implement the multi-lingual version. See if it is possible to remove the attr_readers.